src/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ if (TILES)
                 ${MAIN_CPP}
                 ${MESSAGES_CPP}
                 ${RESOURCE_RC})
+        set_property(TARGET cataclysm-tiles PROPERTY
+        LINK_OPTIONS
+            /MANIFESTINPUT:${CMAKE_SOURCE_DIR}/data/application_manifest.xml
+        )
```

Comment:
This probably won't work for MinGW.

Reply:
The `Makefile` build, as by docs, works.
I will attempt `CMake` without profiles to build under `MSYS2` with `MSYSTEM=mingw64`.